catroid/src/main/java/org/catrobat/catroid/content/bricks/FormulaBrick.java

```diff
@@ -162,10 +162,10 @@ public void storeDataForBackPack(Sprite sprite) {
 	}
 
 	public List<BackPackedListData> getBackPackedListData() {
-		return backPackedListData;
+		return backPackedListData != null ? backPackedListData : new ArrayList<BackPackedListData>();
```

Comment:
It would be better to initialise the member variable at the point of declaration (i.e. 

``` List<BackpackedListData> backPackedListData = new ArrayList<>(); ```

 instead of checking here. you can still overwrite the list at other points and you save yourself this (unnecessary) check.